datareduction/datareducer.cpp

```diff
@@ -3695,7 +3738,14 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
       int myRank;
       MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
       if (myRank == MASTER_RANK) {
-         std::cerr << __FILE__ << ":" << __LINE__ << ": The diagnostic variable " << *it << " is not defined." << std::endl;
+         if (const auto best_match = spell_check(*it)) {
```

Comment:
This uses now an incorrect list of output variables. Here you should check only for accepted diagnostic outputs.

Reply:
True!